src/Makefile.am

```diff
@@ -23,11 +23,15 @@ pytest-cov: $(BUILT_SOURCES) $(DIST_STAMP) $(MANIFESTS)
 if WITH_PYBRIDGE
 INSTALL_DATA_LOCAL_TARGETS += install-python
 install-python:
+	@# HACK: Debian's pip breaks --prefix: https://bugs.debian.org/1035546
+	@# redirect /usr/local to /usr, as merging the trees after install is brittle
+	mkdir -p '$(DESTDIR)$(prefix)'; ln -s . '$(DESTDIR)$(prefix)/local'
```

Comment:
Can we lift this to `debian/rules`?

Reply:
I mean its `make install` rule should create the symlink, call `make install`, then delete the symlink.

Reply:
oh, you mean the symlink, nevermind -- yes, I think I can do that.

Reply:
Done.